content/sdk/10.js/00.ethers/20.v6/01.features.md

```diff
@@ -1,59 +1,50 @@
 ---
-title: ZKsync Era Features
-description:
+title: "zksync Era Features"
+description: "Explore zksync Era Features and Custom Transactions"
+tags: ["zksync", "ethereum", "zksync Era", "transactions", "paymasters"]
 ---
 
-While ZKsync is mostly `ethers`-compatible, it has some differences compared to Ethereum.
-The major of those are:
+zksync Era offers several unique features compared to Ethereum, which require some specific handling. This guide will
+help you understand these differences and how to work with them using the `zksync-ethers` library.
 
-- Account abstraction support (accounts may have near-arbitrary validation logic,
-  and also paymaster support is enabled).
-- Deployment transactions require the contracts' bytecode to be passed in a separate field.
-- The fee system is somewhat different.
+## Key Differences
 
-These require us to extend standard Ethereum transactions with new custom fields.
-Such extended transactions are called EIP712 transactions since [EIP712](https://eips.ethereum.org/EIPS/eip-712)
-is used to sign them. You can look at the internal structure of the EIP712 transactions
-[here](https://docs.zksync.io/zk-stack/concepts/transaction-lifecycle.html#eip-712-0x71).
+::card-group
+::card{title="Account Abstraction" icon="i-heroicons-puzzle-piece"}
+zksync supports account abstraction, allowing accounts to have custom validation logic and enabling paymaster support.
+::
+::card{title="Deployment Transactions" icon="i-heroicons-paper-airplane"}
+Contracts on zksync require bytecode to be passed in a separate field during deployment.
+::
+::card{title="Fee System" icon="i-heroicons-banknotes"}
+The fee system in zksync differs from Ethereum's, necessitating additional fields in transactions.
+::
+::
```

Comment:
Please change this to a bulleted list. The way we currently use cards as links to a page make these look like you can click on them.